pyproject.toml

```diff
@@ -25,7 +25,10 @@ dependencies = [
     "joblib",
     "pytest",
     "tqdm",
-    "multiprocess"
+    "multiprocess",
+    "wandb",
```

Comment:
This is a package dependency?

Reply:
yes

Reply:
Wandb is only required when wandb=True is specified. Is it preferred to add it as a project dependency in this case?